lib/postgrex/extensions/numeric.ex

```diff
@@ -27,6 +27,10 @@ defmodule Postgrex.Extensions.Numeric do
     <<0 :: int16, 0 :: int16, 0xC000 :: uint16, 0 :: int16>>
   end
 
+  def encode_numeric(%Decimal{coef: :inf, sign: sign}) do
+    raise ArgumentError, "Decimal value #{if sign == 1, do: "+", else: "-"}Infinity cannot be represented as numeric type"
```

Comment:
Would you get a better report if you wrote it as:

```elixir
def encode_numeric(%Decimal{coef: :inf} = decimal) do
  raise ArgumentError, "cannot represent #{inspect decimal} as numeric type"
end
```

?

Reply:
Yep, looks better